.prettierrc

```diff
@@ -1,3 +1,4 @@
 {
-  "printWidth": 160
+  "printWidth": 160,
+  "endOfLine": "auto"
```

Comment:
seems like the new standard is to use `lf` https://prettier.io/docs/en/options.html

Reply:
Steve added this because he couldn't build on his Windows machine